library/src/main/scala/scala/scalajs/runtime/IntegerReflectiveCall.scala

```diff
@@ -12,8 +12,10 @@ class IntegerReflectiveCall(value: Int) {
 
   // Methods of scala.Int whose result type is different than in scala.Double
 
+  // scalastyle:off
   def unary_+ : scala.Int = value
   def unary_- : scala.Int = -value
+  // scalastyle:on
```

Comment:
All these seem like scalastyle bugs... It's not like you have an option, right? `unary_+: Int` wont parse...